banyan/src/store/mem_store.rs

```diff
@@ -32,8 +32,8 @@ impl<L: Eq + Hash + Copy> MemStore<L> {
         }))
     }
 
-    pub fn into_inner(self) -> anyhow::Result<FnvHashMap<L, Box<[u8]>>> {
-        let inner = Arc::try_unwrap(self.0).map_err(|_| anyhow!("busy"))?;
+    pub fn into_inner(self) -> Result<FnvHashMap<L, Box<[u8]>>, Error> {
+        let inner = Arc::try_unwrap(self.0).map_err(|_| Error::Busy)?;
```

Comment:
This could be made clearer, for example `Error::MemStore("multiple strong references")`